test/ecto/integration/math_test.exs

```diff
@@ -108,31 +108,34 @@ defmodule Ecto.Integration.MathTest do
       query = from(p in Product, select: fragment("acos(?)", p.price))
 
       # Right now, sqlite will return the acos function as an IEEE float
-      [1.2661036727794992] = TestRepo.all(query)
+      [num] = TestRepo.all(query)
+      assert_in_delta num, 1.266103672779499, 0.000000000000001
```

Comment:
Didn't know about `assert_in_delta`. Good catch by the way.

Reply:
If making the tests a bit looser is ok I'll remove the draft status of the PR.

Reply:
Nah, the approximation stuff is good for floating values. Let's keep it.